Assets/Tests/InputSystem/CoreTests_Actions.cs

```diff
@@ -192,16 +194,30 @@ public void Actions_WhenSeveralBindingsResolveToSameControl_ControlIsAssociatedW
         action3.AddBinding("<Gamepad>/buttonSouth");
         action4.AddBinding("<Gamepad>/buttonSouth"); // Should not be removed; different action.
 
+        action5.AddBinding("<Gamepad>/buttonSouth", interactions: "press(behavior=0)");
```

Comment:
Maybe add a one liner explaining what senario/expected behavior is intended with these scenarios since its not intuitively clear what to expect and why

Reply:
...another option would be to split the test into one-test per scenario